x-pack/test/api_integration/apis/management/index_management/data_streams.ts

```diff
@@ -71,7 +72,7 @@ export default function ({ getService }: FtrProviderContext) {
         expect(dataStreams).to.eql([
           {
             name: testDataStreamName,
-            timeStampField: { name: '@timestamp', mapping: { type: 'date' } },
+            timeStampField: { name: '@timestamp' },
```

Comment:
Can we also remove this `mapping` property from the type? https://github.com/elastic/kibana/blob/master/x-pack/plugins/index_management/common/types/data_streams.ts#L9

I grepped the code for "mapping" and couldn't find any references to it in the context of data streams, so I think we should be fine. TS should give us some safety here too once we update the type and nothing breaks.